.github/workflows/macos-build.yml

```diff
@@ -430,14 +567,14 @@ jobs:
         NINJA_SUMMARIZE_BUILD=1 e build -j $NUMBER_OF_NINJA_PROCESSES
         cp out/Default/.ninja_log out/electron_ninja_log
         node electron/script/check-symlinks.js
+    # Additional Targets: shell_browser_ui_unittests third_party/electron_node:headers third_party/electron_node:overlapped-checker electron:hunspell_dictionaries_zip
```

Comment:
Are we marking this as a follow-up?

Reply:
I actually think all cases except for the `shell_browser_ui_unittests` are handled already here - I'll remove it